packages/veritone-react-common/src/components/ProfileMenu/InnerProfileMenu.js

```diff
@@ -28,6 +28,9 @@ class InnerProfileMenu extends React.Component {
         lastName: string,
         image: string
       }),
+      organization: shape({
+        organizationId: number
+      }),
```

Comment:
Remove unused prop

Reply:
we are using here

Reply:
@kenakaiser I really don't see where can you point me the line that we are using?

Reply:
line 60    
const orgId = get(this.props.user, 'organization.organizationId');

Reply:
Ohh... It's under the user object, got it.